internal/reliabletransport/constants.go

```diff
@@ -0,0 +1,21 @@
+package reliabletransport
+
+const (
+	// Capacity for the array of packets that we're tracking at any given moment (outgoing).
+	RELIABLE_SEND_BUFFER_SIZE = 12
+
+	// Capacity for the array of packets that we're tracking at any given moment (incoming).
+	RELIABLE_RECV_BUFFER_SIZE = RELIABLE_SEND_BUFFER_SIZE
```

Comment:
I remember one of them was 8 and the other 12; could you double check?

Reply:
So, 8 is the max number of packet *ids* that are waiting to be ACKed ... since this is cheap I would ignore.

Now, reliable capacity seems to be fixed at 12: https://github.com/OpenVPN/openvpn/blob/master/src/openvpn/reliable.h#L49

**but** here https://github.com/OpenVPN/openvpn/blob/master/src/openvpn/ssl_pkt.h#L70 it seems that send buffers for TLS is half the size of recv buffers. :confused: